circle.yml

```diff
@@ -2,6 +2,7 @@ general:
   branches:
     ignore:
       - gh-pages
+      - /pull\/.*/
```

Comment:
Thinking about it, can we have this run on pull requests please.  And some how stop the integration tests running for them.